options/options_settable_test.cc

```diff
@@ -433,7 +433,6 @@ TEST_F(OptionsSettableTest, DBOptionsAllFieldsSettable) {
                              "use_direct_io_for_flush_and_compaction=false;"
                              "max_log_file_size=4607;"
                              "advise_random_on_open=true;"
-                             "fail_if_options_file_error=false;"
```

Comment:
Is this there to ensure old options file with fail_if_options_file_error won't return error for DB open?

Reply:
I don't think so. The comments above the test say that the test is to ensure that options can be set through GetDBOptionsFromString().